src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -967,6 +938,39 @@ class LicenceService(
         )
       )
     )
+
+    val sentenceChanges = getSentenceChanges(sentenceDatesRequest, licenceEntity)
+
+    log.info(
+      "Date change flags: LSD ${sentenceChanges.lsdChanged} LED ${sentenceChanges.ledChanged} " +
+        "SED ${sentenceChanges.sedChanged} TUSSD ${sentenceChanges.tussdChanged} TUSED ${sentenceChanges.tusedChanged} " +
+        "isMaterial ${sentenceChanges.isMaterial}"
+    )
+
+    if (!sentenceChanges.isMaterial) return
+
```

Comment:
I've used subscribe here instead of blocking. The email can be sent after we finish with the update request.